src/coreclr/nativeaot/System.Private.CoreLib/System.Private.CoreLib.csproj

```
@@ -3,14 +3,12 @@
   <!-- Note that various places in SPCL depend on this resource name i.e. TplEventSource -->
   <ItemGroup Label="Embedded Resources">
     <EmbeddedResource Include="src\Resources\Strings.resx">
+      <LogicalName>System.Private.CoreLib.Strings.resources</LogicalName>
```

```suggestion
      <ManifestResourceName>System.Private.CoreLib.Strings</ManifestResourceName>
```
Setting ManifestResourceName should cut off the problem closer to where it gets created.

---

I'm looking into moving the CoreLib project into src like they do in the libraries partition. The existing structure is inspired by CoreCLR's CoreLib, but it's inconsistent.

The test failures are because we can't find CoreLib's resource strings because of this hack :(